R/progress-bars.R

```diff
@@ -1,18 +1,18 @@
-
 #' Progress bars in purrr
 #'
-#' Some purrr functions have a `.progress` argument, that you can use to
-#' create a progress bar.
+#' @description
+#' purrr's map functions come have a `.progress` argument, that you can use to
+#' create a progress bar. `.progress` can be:
 #'
-#' Possible values for `.progress`:
+#' * `FALSE`, the default: purrr does not create a progress bar.
+#' * `TRUE`: purrr creates a basic unnamed progress bar.
```

Comment:
```suggestion
#' * `FALSE`, the default: does not create a progress bar.
#' * `TRUE`: creates a basic unnamed progress bar.
```

Matches format of `A string:` bullet